src/TestUtils/src/UITest.Appium/Actions/AppiumAndroidThemeChangeAction.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Diagnostics;
+using UITest.Core;
+
+namespace UITest.Appium
+{
+	public class AppiumAndroidThemeChangeAction : ICommandExecutionGroup
+	{
+		const string SetLightTheme = "setLightTheme";
+		const string SetDarkTheme = "setDarkTheme";
+
+		readonly List<string> _commands = new()
+		{
+			SetLightTheme,
+			SetDarkTheme
+		};
+
+		public CommandResponse Execute(string commandName, IDictionary<string, object> parameters)
+		{
+			if (commandName == SetLightTheme)
+			{
+				ExecuteAdbCommand($"adb shell cmd uimode night no");
```

Comment:
Nice usage of adb. Could be nice for older Android versions. From Android 10 and newer you can use https://github.com/appium/appium-uiautomator2-driver?tab=readme-ov-file#mobile-setuimode

**mobile: setUiMode**
A thin wrapper over adb shell cmd uimode CLI.

Reply:
Done :)